test/MUXControls.Test/Infra/TestHelpers.cs

```diff
@@ -163,6 +163,15 @@ public TestSetupHelper(ICollection<string> testNames, string languageOverride =
 
                         Log.Comment("__TestContentLoadedCheckBox checkbox checked, page has loaded");
 
+
+                        ToggleButton tb = new ToggleButton(FindElement.ById("__InnerFrameInLabDimensions"));
```

Comment:
I am slightly concerned that this might interfere with CI runs in unexpected ways. Though, in theory, now running tests on developer machines and on the CI should be more similar, so this probably doesn't make a difference than a few milliseconds more test run time.